core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -910,4 +922,10 @@ public String toStringResponseOnly(Function<HttpHeaders, HttpHeaders> headersSan
             this.interestedFlags = interestedFlags;
         }
     }
+
+    // This is millisecond precision on Java 8, but it's the best we can do.
+    private static long currentTimeMicros() {
+        Instant now = Clock.systemUTC().instant();
+        return TimeUnit.SECONDS.toMicros(now.getEpochSecond()) + TimeUnit.NANOSECONDS.toMicros(now.getNano());
```

Comment:
How about using `PlatformDependent.javaVersion()` and just use `System.currentTimeMillis()` up to Java 8?